tests/gtests/sycl/api/test_stream.cpp

```diff
@@ -98,6 +98,22 @@ TEST_F(sycl_stream_test, BasicInterop) {
 #endif
 }
 
+TEST_F(sycl_stream_test, BasicInteropCPU) {
+    cl_device_id cpu_ocl_dev = find_ocl_device(CL_DEVICE_TYPE_CPU);
+    SKIP_IF(!cpu_ocl_dev, "CPU device not found.");
```

Comment:
These two lines seem unnecessary. What do you try to catch here that would not be caught by the `SKIP_IF` below?

Reply:
same logic as in other test: https://github.com/oneapi-src/oneDNN/blob/9e039c70c1ac1b5d012265d7b9229e1a98c1b442/tests/gtests/sycl/api/test_stream.cpp#L120

I guess this two lines can skip the test for non-intel CPU device if `find_ocl_device` can check the vendor. Or just handle the ocl runtime error if the UMD driver is not well configured. @densamoilov any suggestions here?

Reply:
I think if the OpenCL doesn't support the CPU then creating the queue below will throw an exception. I think we should keep these lines.